ReactAndroid/src/main/java/com/facebook/react/bridge/JSDebuggerWebSocketClient.java

```diff
@@ -118,7 +118,6 @@ public void executeApplicationScript(
       for (String key : injectedObjects.keySet()) {
         jg.writeObjectField(key, injectedObjects.get(key));
       }
-      jg.writeEndObject();
```

Comment:
If that's the case, then there are other places in this file sendMessage is called as well that have the same issue. Instead of having a contract where sendMessage closes the JSON object ( >____> ), we should leave the `jg.writeEndObject();` here and remove it from sendMessage. Make sure to have a test plan that tests using the JS debugger though!